dist.Dockerfile

```diff
@@ -8,6 +8,10 @@ ARG gid=10001
 COPY . /var/cartography
 WORKDIR /var/cartography
 
+RUN apt-get update --fix-missing
+RUN apt --allow-unauthenticated  update -y
+RUN apt-get install vim telnet curl gcc -y
```

Comment:
Wait, I hadn't noticed this before, why is vim, telnet, curl and gcc being added?